src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Added the `dispatcher` option to `debug:event-dispatcher`
  * Added the `event_dispatcher.dispatcher` tag
  * Added `assertResponseFormatSame()` in `BrowserKitAssertionsTrait`
+ * Added tag `assets.package` to register asset packages
```

Comment:
s/Added/Add according to our recent policy clarification (but leave the older lines as is)